<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>331</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Nama :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Email :</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Password :</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Role:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Jabatan :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="namaLineEdit"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="emailLineEdit"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="passwordLineEdit"/>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="roleComboBox"/>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="jabatanComboBox"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
